CONTRIBUTING.md

```diff
@@ -30,13 +30,21 @@ The public key can be found on [coblox.tech](https://coblox.tech/security_coblox
 
 Thank you for taking the time to make COMIT safer!
 
-
 ## Feature Requests
 
 To request a change on the way COMIT works, please head over to the [RFCs repository](https://github.com/comit-network/rfc) and checkout the [RFC contributing guidelines](https://github.com/comit-network/rfc/blob/master/CONTRIBUTING.md).
  
 For changes related to our implementation, please [create a feature request](/issues/new/choose) and follow the GitHub template.
 
+## Fixing an issue
+
+If you wish to pick up an existing issue and resolve it, please be aware of the following labels:
+- [`groomed`](https://github.com/comit-network/comit-rs/labels/groomed): it has been discussed and should contain enough details to get started
+- [Unlabelled](https://github.com/comit-network/comit-rs/issues?q=is%3Aopen+no%3Alabel): it has not yet been reviewed for design, please avoid
+- [`icebox`](https://github.com/comit-network/comit-rs/labels/icebox): it is not the right time to proceed with this issue, please do not pick it
+- [`sprint-backlog`](https://github.com/comit-network/comit-rs/labels/sprint-backlog): it will be picked up in our next sprint, please do not pick it
+- [`candidate-contribution`](https://github.com/comit-network/comit-rs/labels/candidate-contribution): it has been groomed to be used as part of our hiring process, please do not pick it; [keen to join?](https://tenx.workable.com/jobs/689268)
```

Comment:
```suggestion
- [`candidate-contribution`](https://github.com/comit-network/comit-rs/labels/candidate-contribution): it has been groomed to be used as part of our hiring process [keen to join?](https://tenx.workable.com/jobs/689268)
```
I think we should be happy for any contribution and we should not stop someone from doing tickets :)

Reply:
ok are you thinking the same about `sprint-backlog` then?